ui/de-lib/src/main/java/org/iplantc/de/client/util/StaticIdHelper.java

```diff
@@ -0,0 +1,65 @@
+package org.iplantc.de.client.util;
+
+import com.google.common.base.Preconditions;
+import com.google.gwt.dom.client.Element;
+import com.google.gwt.dom.client.NodeList;
+
+import com.sencha.gxt.core.client.dom.XElement;
+import com.sencha.gxt.widget.core.client.grid.ColumnHeader;
+import com.sencha.gxt.widget.core.client.grid.ColumnModel;
+import com.sencha.gxt.widget.core.client.grid.Grid;
+
+/**
+ * @author aramsey
+ */
+public class StaticIdHelper {
+
+    private static StaticIdHelper INSTANCE;
+
+    StaticIdHelper() {
+
+    }
+
+    public static StaticIdHelper getInstance(){
+        if(INSTANCE == null) {
+            INSTANCE = new StaticIdHelper();
+        }
+
+        return INSTANCE;
+    }
+
+    /**
+     * Sets the static IDs for the column headers in a grid
+     * This does not work if called simply within a view's onEnsureDebugId method
+     * as the headers will not have been rendered yet.
+     * This can be called successfully using the grid's ViewReadyHandler (which can
+     * then be added to the grid within the onEnsureDebugId method)
+     * @param baseID
+     * @param grid
+     * @param cm
+     */
+    public void gridColumnHeaders(String baseID, Grid<?> grid, ColumnModel<?> cm) {
```

Comment:
QA asked specifically for IDs on each column header, and this was the best I could come up with.  From what I saw, Sencha's Grid/GridView and ColumnModel/ColumnHeader classes were too intertwined to find an easy way to extend a single class and set those IDs for each column header.

Sencha's ColumnHeader appearance gives me access to the header style class name, which I then use to find the whole column header's `table` and its inner `td` elements which are each individual column header.